test/common/testlib.py

```diff
@@ -634,8 +638,6 @@ def allow_restart_journal_messages(self):
 
     def allow_authorize_journal_messages(self):
         self.allow_journal_messages("cannot reauthorize identity.*:.*unix-user:admin.*",
-                                    "Error executing command as another user: Not authorized",
-                                    "This incident has been reported.",
```

Comment:
Hmmm, it seems that we add them to the default set to be ignored. That should mean they shouldn't need to be listed anywhere else. Or am I misunderstanding?